src/content/extension.js

```diff
@@ -1,24 +1,261 @@
-import GitHub from "./github";
-import LeetCode from "./leetCode";
-import StackOverflow from "./stackOverflow";
+/* globals chrome */
 
+import prettier from "prettier/standalone";
 import Storage from "./storage";
+import { PARSERS, PARSERS_LANG_MAP } from "./parsers";
 
-const extensions = [GitHub, StackOverflow, LeetCode];
+const STACKEXCHANGE_SITES = [
```

Comment:
Inlined all this for now but we can always create more abstractions in the future if this becomes more complex